docs/docs/conceptual-guide.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Conceptual Guide
-overview: true
+título: Conceptual Guía
+visión de conjunto: cierto
 ---
 
-Read high-level overviews of the Gatsby approach.
+Leer alto-nivel visión de conjunto de el Gatsby enfoque.
```

Comment:
This a literal translation from a poor word translator. It doesn't even make any sense in spanish :/

Reply:
Can you please tell me what changes should i do ?